source/opt/def_use_manager.h

```diff
@@ -63,6 +63,9 @@ class DefUseManager {
   // const overload for ForEachInst().
   void AnalyzeDefUse(ir::Module* module);
 
+  // Analyzes the defs and uses in the given |inst|.
+  void AnalyzeInstDefUse(ir::Instruction* inst);
```

Comment:
Need more in the blurb now.  (Or maybe you always needed more in the blurb...)
What if I analyze the same instruction multiple times?  What if I've analyzed some instructions but not others yet.

Basically, the consistency of the data in this manager is in question. Before you could guess that it had no data until AnalyzeDefUse(module) was called.  Then after that you knew it had all the information.  But now that we can do things incrementally, it seems a bit up in the air.